src/MassTransit.MessagePack/Serialization/MessagePackMessageSerializerContext.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace MassTransit.Serialization;
+
+using System;
+using System.Collections.Generic;
+using MessagePack;
+using MassTransit.Internals.Json;
+
+
+public class MessagePackMessageSerializerContext : BaseSerializerContext
```

Comment:
Style, but MassTransit always has a line break after the `:` and before the first basetype/interface.

Reply:
I have fixed all occurrences of this.